### PR TITLE
Sveltekit tutorial

### PR DESCRIPTION
**Pull request by alvinometric**
Simple Sveltekit tutorial

### CODE REVIEW

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+## Conclusion
+
+That's it, you have a Sveltekit app with fleature flags. More precisely, you've learned:
+
+- How to make a habit tracking app with Sveltekit
+- How to add feature flags and how to add them to a fullstack app
+- The different approaches to feature flagging on a static vs SSR context
+
+Enjoy, see you soon.
```
Comment (nnennandukwe) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:267]: I'd suggest removing this line for now and maybe we can replace it with pointing devs to a link to other useful docs in the future. Example:

Take a look at our [insert link] documentation for a deeper dive into advanced use cases with Sveltekit.

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
+---
+title: How to Implement Feature Flags in Sveltekit using Unleash
```
Comment (nnennandukwe) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:2]:
```suggestion
title: How to Implement Feature Flags in SvelteKit
```

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+npm create svelte@latest habits
+```
+
+We'll need a few more dependencies, let's install them now so it's out of the way.
+
+```
+npm i date-fns @unleash/proxy-client-svelte unleash-client
+```
+
+## Let's make a basic habits app
```
Comment (nnennandukwe) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:24]:
```suggestion
## Create a basic habits app
```

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+## Setup
+
+Create a skeleton Sveltekit project named "habits".
+
+```
+npm create svelte@latest habits
+```
+
+We'll need a few more dependencies, let's install them now so it's out of the way.
```
Comment (nnennandukwe) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:18]:
```suggestion
We'll need a few more dependencies. You can install these in one command below:
```

**Review by nnennandukwe — CHANGES_REQUESTED** [review]
Great work @alvinometric ! 👏 

I've added quite a few comments that mostly cover:
- structure (sentences & layout of the doc)
- any clarification needed on the "what" and "how" in your steps

Those comments are to make sure we can be aligned on how we want to convey information to developers and ensure we're using a good structure that is easy to follow and easily discoverable when people search for SvelteKit feature flag things (while still using our own voice in the tutorials). 

Additional suggestions:
- I'd recommend replacing references of "Sveltekit" with "SvelteKit" in the doc. Whichever one is the official capitalization for the name. To my knowledge, it's "SvelteKit".
- Include an architectural diagram of how the SvelteKit app will be hooked up to the SDK and Unleash. I personally use Excalidraw for the diagram and you can use this as reference. Replace "React app" with "SvelteKit app" and the feature flag on-and-off states specific to this tutorial like so:
Screenshot 2024-03-13 at 1 37 42 PM

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+</tr>
+```
+
+And now here's our full app in all its glory, basically a table with checkboxes.
+
+![Pasted image 20240202174256.png](./20240202174256.png)
+
+## Adding habits and premium features
+
+Now this is not a very user-friendly app. Let's add a few things:
```
Comment (nnennandukwe) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:147]:
```suggestion
We have the basics of the app set up, but we could make it more user-friendly. Let's add some more functionality:
```

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+  // src/routes/+page.svelte
+  import App from '../lib/App.svelte';
+</script>
+
+<App />
+```
+
+And we finish the basic app with a component for each habit that be checked on and off.
+
+```svelte
```
[the remark this answers is not in the document]
Comment (alvinometric) [reply website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:95]: It's the first comment in the file

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+      />
+      {format(date, 'MMM do')}
+    </td>
+  {/each}
+</tr>
+```
+
+Now we have a fully functioning Svelte app in all its glory! Essentially, it's a table with checkboxes.
+
+![Pasted image 20240202174256.png](./20240202174256.png)
```
Comment (thomasheartman) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:143]: This image needs an update as described in my general review comment.

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+  title="A maximum number of habits on the free tier"
+  src="/media/habits.mp4"
+  controls> </video>
+
+## Showing a different component based on the feature flag
+
+On to the main topic, adding feature flags.
+
+Go to your Unleash dashboard, and create new project (you're welcome to use the default project here).
+![alt text](./proj.png)
```
Comment (thomasheartman) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:218]: Same here! The alt text appears to have been started but not finished?

File: website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+
+Next, create a feature flag called `maxHabitsIncreased`.
+
+![alt text](./feat.png)
```
Comment (thomasheartman) [inline website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md:222]: And the same for this one.

**Review by thomasheartman — COMMENTED** [review]
Hey, Alvin! Nice work 🙌🏼 

Looks like Nnenna already gave you a very thorough and great review (👏🏼 👏🏼 👏🏼 ) and is probably a much better person for reviewing the content than I am.

However, I wanna chuck in a few comments on images, if you don't mind:

1. Please make sure they have a proper alt text, so that it's clear for people who can't see the image what it portrays. I'm assuming you're familiar with alt texts (it appears you've put in placeholders in a few of the places), but regardless, I have found this [blog post on alt texts from axess lab](https://axesslab.com/alt-texts/) to be a very useful primer/refresher on writing them.
2. We usually put images in the static/img directory and not right next to the doc. Of course, we can consider changing that, but we should probably do that in one go.

**Review by nnennandukwe — APPROVED** [review]

**Pull request merged**